apps/prairielearn/src/lib/opsbot.ts

```diff
@@ -1,4 +1,4 @@
-import detectMocha from 'detect-mocha';
+import detectMocha = require('detect-mocha');
```

Comment:
For my own learning, what does the `require` do here?

Reply:
Docs: https://www.typescriptlang.org/docs/handbook/modules/reference.html#export--and-import--require

This is necessary to import modules that are declared as `export = ...` with our current TypeScript configuration. Specifically, we don't currently have `allowSyntheticDefaultImports: true` because our runtime (Node CJS) doesn't support that. If we try to `import detectMocha from 'detect-mocha'`, we'll get the following TypeScript error:

> This module is declared with 'export =', and can only be used with a default import when using the 'allowSyntheticDefaultImports' flag.

This will all go away once we're using native ESM and can set `allowSyntheticDefaultImports`.